hw1/02-palindrome.js

```diff
@@ -1,3 +1,41 @@
-const elem = document.querySelector('input');
+// Declarations
+const elem = document.querySelector("input");
+elem.addEventListener("change", handleInput); // get input on enter
```

Comment:
Interesting that you changed this to check upon entering. I like this change

Reply:
Thank you!  I noticed it was inputting on each new character, so I wanted to adjust that to only input on enter.